src/content/docs/browser/browser-monitoring/browser-pro-features/session-replay.mdx

```diff
@@ -272,7 +240,7 @@ In the UI, you can find session replays in two places:
 <CollapserGroup>
```

Comment:
I'm inclined to want to move the troubleshooting section to the browser monitoring troubleshooting category (the doc is already so long!) I think it will make it easier for people to find the information in these two collapsers.

https://deploy-preview-16499--docs-website-netlify.netlify.app/docs/browser/new-relic-browser/troubleshooting/troubleshoot-ajax-data-collection/